scripts/tts_dataset_files/ipa_cmudict-0.7b_nv23.01.txt

```diff
@@ -134824,3 +134822,4 @@ TUESDAY(1)  ˈtuzdi
 TV  ˈtiˈvi
 WEDNESDAY  ˈwɛnzˌdeɪ
 WEDNESDAY(1)  ˈwɛnzdi
+WHATEVER'S(1)  ˌhwʌˈtɛvɚz
```

Comment:
Can you add a comment that we removed `WHATEVER'S(1)` and `WHATEVER(1)`? Instead of this?

Reply:
oh, do you mean add a comment in the file in the "notes" part?

Reply:
Yes

Reply:
Ok. I added to notes. Thanks